InterfaceMATLAB/tsgCoreTests.m

```diff
@@ -445,6 +445,44 @@ function tsgCoreTests()
 end
 tsgDeleteGrid(lGrid);
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%                     tsgMakeFourier()                             %%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% test transform
+[lGrid, p] = tsgMakeFourier('_tsgcoretests_lgrid', 3, 1, 'level', 1, [3.0 5.0; -7.0 -6.0; -12.0 17.0]);
+if ((abs(max(p(:, 1)) - 13.0/3.0) > 1.E-11) || (abs(min(p(:, 1)) - 3.0) > 1.E-11) ...
+    || (abs(max(p(:, 2)) + 19.0/3.0) > 1.E-11) || (abs(min(p(:, 2)) + 7.0) > 1.E-11) ...
+    || (abs(max(p(:, 3)) - 22.0/3.0) > 1.E-11) || (abs(min(p(:, 3)) + 12.0) > 1.E-11))
+    error('Mismatch in tsgMakeFourier: transform');
+end
+[w, p] = tsgGetQuadrature(lGrid);
+if ((abs(norm(sum(w)) - 58.0) > 1.E-11) || (abs(max(p(:, 1)) - 13.0/3.0) > 1.E-11) ...
+    || (abs(min(p(:, 1)) - 3.0) > 1.E-11) || (abs(max(p(:, 2)) + 19.0/3.0) > 1.E-11) ...
+    || (abs(min(p(:, 2)) + 7.0) > 1.E-11) || (abs(max(p(:, 3)) - 22.0/3.0) > 1.E-11) ...
+    || (abs(min(p(:, 3)) + 12.0) > 1.E-11))
+    error('Mismatch in tsgMakeFourier: getQuadrature');
+end
+
+% correctness of 1-D
+[lGrid, pf] = tsgMakeFourier('_tsgcoretests_lgrid', 1, 1, 'level', 2);
```

Comment:
* why use `pf` as opposed to just `p`
* also, why use `bmark` as opposed to `tp`, I use `tp` in tests to indicate `true points`
* consistent variable names make the code easier to read